app/utils/prepareAnswer.js

```diff
@@ -90,9 +90,20 @@ function prepareDailyStatsAnswer(body, n) {
     return data
 }
 
+function prepareTestingResourceAnswer(stateResources) {
+    var data = `\n\n\u{1F9EA} COVID-19 Testing Labs in ${stateResources[0].state}, found ${stateResources.length} results\n`
```

Comment:
`test hello` will give error here, its good to check if `stateResources` is not `undefined` before actually using it.